container/image.bzl

```
@@ -459,8 +462,10 @@ def _impl(
             null_cmd = null_cmd,
         )
 
-    # Construct a temporary name based on the build target.
-    tag_name = "{}:{}".format(_repository_name(ctx), name)
+    # Construct a temporary name based on the build target. This is the name
+    # of the docker container.
+    final_tag = tag_name if tag_name else name
+    tag_name = "{}:{}".format(_repository_name(ctx), final_tag)
```

```suggestion
    # Construct the name of the docker container
    container_name = "{}:{}".format(_repository_name(ctx), tag_name if tag_name else name)
```

How about calling the variable `container_name`? From the [syntax](https://ktomk.github.io/pipelines/doc/DOCKER-NAME-TAG.html), "tag_name" is just the part after `:`. The full string is "container".